bbc/simorgh
# WSTEAM1-1315 - Hide timestamp on Live TV assets

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-1315

Overall changes
======
- Adds function to check if asset being viewed is a 'live' asset and the only one on the page. If so, hide the timestamp

Code changes
======
- Adds `checkIsLiveMedia` function to determine whether or not to hide the timestamp on MAP pages
- Renames `isMap` to `isCpsMap` for clarity
- Adds Storybook story for LiveTV asset to track visual regressions

Testing
======
1. Visit http://localhost:7080/arabic/media-49522519?renderer_env=live
2. Confirm the timestamp is not visible
3. Visit http://localhost:7080/hausa/articles/cdx6ddxw755o?renderer_env=live
4. Confirm the timestamp is visible

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
